### DISCUSSION TITLE
Trigger each days and only once

### DISCUSSION THREAD

**Discussion by Zefix0509** [quartznet/quartznet#1827] [Q&A] [answered]
Hello

Since the last few weeks, I'm trying to make a trigger that will run my job each days of the week only one time at a specific time then will wait for the next day to run the same job again and never stop over the year ?

I can choose which days of the week I want my job to be executed too so we can't assume that it will be each 24 hour.

Thanks for the help!

**Comment by lahma** [comment, marked as answer]
Have you tried http://www.cronmaker.com/ , check the weekly tab.

**Reply by Zefix0509** [reply]
Oh I didn't know this website thank you so much..

but does my job will only get executed one time like if I set it to start at 8am my job will execute then wait for the next day?

**Reply by lahma** [reply]
Yes, it will fire at 8am. Then whenever the next fire time is. There's a [good tutorial on Quartz website](https://www.quartz-scheduler.net/documentation/quartz-3.x/tutorial/) site too.

**Reply by Zefix0509** [reply]
Thanks I've read the tutorial and tried but my code didn't work

And how will I add my job and trigger to my scheduler ? 

scheduler.addjob(myjob);
scheduler.schedulejob(mytrigger) ;
scheduler.triggerjob(myjobkey); 

Which one is the good one ?

**Reply by lahma** [reply]
Now it feels that you want me to write your code. Please see the examples and study the API.

**Reply by Zefix0509** [reply]
thanks for your help great support...
